### PR TITLE
Fix the error from PowerShellCodeCheck_PSScriptAnalyzer Job

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

**Comment by LiliDeng** [comment]
@lizzha I run functional and regression testing, passed against Hyper-V and Azure platforms.

File: Libraries/Framework-Azure.psm1 [hunk 1]
```diff
@@ -668,9 +664,13 @@ Function FinishLogReport([bool]$isFinal=$True)
 	if($isFinal)
 	{
 		$global:junitReport = $null
+		LogMsg "$global:junitReport"
```
Comment (lizzha) [inline Libraries/Framework-Azure.psm1]: is this change expected?

Comment (lizzha) [reply Libraries/Framework-Azure.psm1]: Per offline talk, these "LogMsg"s fix the warning of variables not used

File: Testscripts/Windows/CORE-VerifyHeartBeat.ps1 [hunk 2]
```diff
@@ -31,10 +31,11 @@ function Main {
         $tokens = $p.Trim().Split('=')
         if ($tokens.Length -ne 2) {
             LogMsg "Warn: test parameter '$p' is being ignored because it appears to be malformed"
-            continue
+            continue
         }
         if ($tokens[0].Trim() -eq "TC_COVERED") {
             $TC_COVERED = $tokens[1].Trim()
+            LogMsg "TC_COVERED: $TC_COVERED"
```
Comment (lizzha) [inline Testscripts/Windows/CORE-VerifyHeartBeat.ps1]: is this $TC_COVERED needed?

Comment (LiliDeng) [reply Testscripts/Windows/CORE-VerifyHeartBeat.ps1]: @lizzha Removed.

**Pull request merged**